example/helm/values.yaml

```
@@ -35,10 +36,24 @@ gateway:
       requests:
         cpu: 150m
         memory: 150Mi
+    hpa:
+      enabled: false
+    ingress:
+      # if enabled is true, vald-lb-gateway can be connected through Kubernetes ingress to the external.
```

```suggestion
      # if enabled is true, vald-lb-gateway can be connected through Kubernetes ingress from the external network.
```

---

```suggestion
      # if enabled is true, vald-lb-gateway can be connected through Kubernetes ingress to the external network.
```
In this scene, I think it'd be better.

---

LB-gateway is connected from the external network so it should be `from`

---

I think lb-gateway is connecting to the external network lol
cuz, external is not k8s cluster, and the main character is vald cluster.

---

@kevindiu it doesn't matter which one but my thought is ⬆️

---

@vankichi 
The connection to connect to the lb-gateway is established from the client, so I think is should be `from`.
If we are using `to` here, it means that the connection is established from the lb-gateway to the external network, which is not correct.